src/render-html/messageAsHtml.js

```diff
@@ -72,12 +112,36 @@ const messageBody = ({
   reactions: ReactionType[],
   realmEmoji: ReactionType,
   timeEdited: Date,
-}) => `
+}) => {
+  let val = null; 
+  // Regex to scan out image URLs
+  const regex = /<img src="(.*)">/g; 
```

Comment:
Regex to catch img tag urls

Reply:
There is a method `isUrlAnImage` in `utils/url` you can use that instead with the regex to scan out urls.